src/mds/MDLog.cc

```diff
@@ -656,6 +659,10 @@ void MDLog::try_to_commit_open_file_table(uint64_t last_seq)
   }
 }
 
+bool MDLog::is_oversegmented() const {
```

Comment:
```suggestion
bool MDLog::is_oversegmented() const {
```

"oversegmented" to me means something like "over fragmented"? I think just `::is_trimming_slow` would perhaps be better.

Reply:
Changed.